lib/takeaway.rb

```diff
@@ -0,0 +1,119 @@
+require "Pry"
+
+
+ class Takeaway
+
+  attr_reader :text, :menu, :order  
+
+  def initialize
+      @menu = Menu.new 
+      @order = Order.new 
+  end 
+ 
+   def greeter 
```

Comment:
This is a great idea to give the user all the different options at the start. Wish I'd done this!